doc/release/1.11.0-notes.rst

```diff
@@ -191,8 +191,18 @@ all of these possible values are unicode- and case-insensitive).
 Such behaviour will not be allowed in future releases.
 
 Random number generator in the ``testing`` namespace
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Python standard library random number generator was previously exposed in the
 ``testing`` namespace as ``testing.rand``. Using this generator is not
 recommended and it will be removed in a future release. Use generators from
 ``numpy.random`` namespace instead.
+
+Random integer generation on a closed interval
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+In accordance with the Python C API, which gives preference to the half-open
+interval over the closed one, ```np.random.random_integers``` is being
+deprecated in favor of calling ```np.random.randint``` with the arguments
+appropriately adjusted. Work has already been done to begin phasing out
```

Comment:
Should probably mention that `random_integers` will not be removed anytime soon. Also, instead of saying that `random_integers` lacks the new features, point out that `randint` has new features not available in `random_integers`. Make it an endorsement of `randint`, not a put down of `random_integers`.